### PR TITLE
Wishlist Screen

### PR DESCRIPTION
**Pull request by horsemaker**
- Added Wishlist Screen

recording

### CODE REVIEW

File: screens/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,308 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (AtulHande03a) [inline screens/wishlist.html:6]: author meta tag

**Review by AtulHande03a — APPROVED** [review]

**Review by MarkVed17 — APPROVED** [review]
Looks good to me!

File: screens/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+            <li class="navbar-link">
+              <a href="#">About Us</a>
+            </li>
+            <li class="navbar-link">
+              <a href="product-listing.html">Products</a>
+            </li>
+          </ul>
+        </section>
+        <section class="navbar-actions">
+          <!-- <span class="material-icons navbar-action navbar-icon"> search </span> -->
```
Comment (t007rushi) [inline screens/wishlist.html:39]: Don't forget unwanted comments

File: stylesheets/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
+.wishlist {
+  margin-top: 7.5rem;
+  scroll-margin-top: 7.5rem;
+  padding: 0 5rem;
+  display: flex;
+  gap: 1rem;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```
Comment (t007rushi) [inline stylesheets/wishlist.css:9]: Utility class...🧐

**Review by t007rushi — APPROVED** [review]
Good to merge with micro changes

File: screens/wishlist.html (hunk 1 above)
Comment (horsemaker) [reply screens/wishlist.html:6]: Have noted it, will include them in the next misc PR

File: screens/wishlist.html (hunk 2 above)
Comment (horsemaker) [reply screens/wishlist.html:39]: I plan to add it later, if possible

File: stylesheets/wishlist.css (hunk 3 above)
Comment (horsemaker) [reply stylesheets/wishlist.css:9]: In the next misc PR

**Pull request merged**